docs/installation.md

```diff
@@ -29,3 +29,24 @@ As well, you may wish to customize your installation in some other fashion. or e
 - [Debian Package Installation Notes](http://progrium.viewdocs.io/dokku/getting-started/install/debian)
 - [Vagrant Installation Notes](http://progrium.viewdocs.io/dokku/getting-started/install/vagrant)
 - [Advanced Install Customization](http://progrium.viewdocs.io/dokku/advanced-installation)
+
+### VMs with only up to 512mb memory
```

Comment:
Maybe worth moving this to `Advanced Install Customization` and listing 1GB as an install requirement above (linking to the section in advanced install customization of course).